.github/workflows/ci_packages.yml

```diff
@@ -61,3 +61,19 @@ jobs:
       - name: 'koch, Run CI'
         shell: bash
         run: . ci/funs.sh && nimInternalBuildKochAndRunCI
+
+      - name: 'Run SSL server tests'
```

Comment:
c_packages should be reserved for nimble packages though; why not handle this in azure-pipelines.yml instead which is for internal CI code?